libtirpc.sh

```diff
@@ -0,0 +1,33 @@
+package: libtirpc
+version: "%(tag_basename)s"
+tag: "libtirpc-1-1-4"
+source: git://git.linux-nfs.org/projects/steved/libtirpc.git
+build_requires:
+  - CMake
```

Comment:
A small change: it does not seem the module requires CMake (I see `./configure` below) so you can drop this dependency. Just check whether it depends on stuff like `autotools` and add it properly (we have a recipe for that already).

Reply:
right! copy-paste artifacts, done